process_jartic_archives.py

```
@@ -0,0 +1,280 @@
+        parts = archive.stem.split('_')
+        if len(parts) >= 4:
+            try:
+                year = int(parts[2])
+                month = int(parts[3])
+                archive_date = datetime(year, month, 1, tzinfo=timezone.utc)
+                
+                # Check if archive month overlaps with requested date range
+                if archive_date.year == args.start.year and archive_date.month >= args.start.month:
+                    filtered_archives.append(archive)
+                elif archive_date.year == args.end.year and archive_date.month <= args.end.month:
+                    filtered_archives.append(archive)
```

Ruff (0.12.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)